packages/expo-image/android/src/main/java/expo/modules/image/ExpoImageView.kt

```diff
@@ -139,6 +143,7 @@ class ExpoImageView(
       progressInterceptor.registerProgressListener(sourceToLoad.toStringUrl(), eventsManager)
       eventsManager.onLoadStarted()
       requestManager
+        .asDrawable()
```

Comment:
Calling `requestManager.load()` is a short form of `requestManager.asDrawable().load()` - I wanted to explicitly specify we want a `Drawable`.

Other options would be `asFile()`, `asGif()` or `asBitmap()` - unfortunately the last one is probably not supported by all file formats